ask-sdk-core/ask_sdk_core/api_client.py

```diff
@@ -27,7 +27,7 @@
 from .exceptions import ApiClientException
 
 if typing.TYPE_CHECKING:
-    from typing import Callable, Dict, List, Tuple
+    from typing import Callable, Dict, List, Tuple, Optional
```

Comment:
I don't think `Optional` is used anywhere in the file. Why was this added?

Reply:
Yep, I have added another commit removing all unused typing libraries.
Thank you for noticing.